content/docs/reference/config/annotations/index.html

```diff
@@ -0,0 +1,290 @@
+---
+title: Annotations
+description: Istio Annotations that can be applied to other Mesh resources.
+location: https://istio.io/docs/reference/config/annotations.html
```

Comment:
Please remove location, layout, generate, and number_of_entries, these are only used for the auto-generated docs. You should also have a weight: value to control the sorting order within the contanining directory.

Any reason you chose to do this doc in HTML instead of markdown?

Reply:
I would like to do this doc in markdown.  I based this page on its siblings in _content/docs/reference_.  Is there a better example that I could follow?

Reply:
All the other docs in istio.io are in markdown. The only HTML material is what the proto conversion tool generates.

Check out https://preliminary.istio.io/about/contribute/creating-and-editing-pages/

Reply:
@geeknoid Let me be concrete.   https://preliminary.istio.io/docs/reference/config/authorization/istio.rbac.v1alpha1/ includes the text "A ServiceRole specification includes a list of rules".  I searched for that text on my system and found that text in just two places, _istio.io/content/docs/reference/config/authorization/istio.rbac.v1alpha1/index.html_ and _api/rbac/v1alpha1/rbac.proto_.  For annotation we don't have a .md and we don't have a .proto.

Please advise.